src/main/java/de/unijena/cheminf/mortar/controller/MainViewController.java

```diff
@@ -440,10 +458,11 @@ private void closeWindowEvent(WindowEvent anEvent) {
         anEvent.consume();
     }
     //
+
     /**
-     * Loads molecule file and opens molecules tab.
+     * Opens a file choose, loads the chosen file and opens molecules tab
      *
-     * @param aParentStage Stage where to open the file chooser dialog
+     * @param aParentStage aParentStage Stage where to open the file chooser dialog
      */
     private void importMoleculeFile(Stage aParentStage) {
```

Comment:
Maybe rename this method to make the difference more clear. "chooseAndImportMoleculeFile()"?

Reply:
Therefore the javadoc, but if it makes you happy